src/platform/linux/input/inputtino_gamepad.cpp

```diff
@@ -258,7 +258,9 @@ namespace platf::gamepad {
           state = inputtino::PS5Joypad::BATTERY_FULL;
           break;
       }
-      std::get<inputtino::PS5Joypad>(*gamepad->joypad).set_battery(state, battery.percentage);
+      // Battery values in Moonlight are in the range [0, 0xFF (255)]
+      // Inputtino expects them as a percentage [0, 100]
+      std::get<inputtino::PS5Joypad>(*gamepad->joypad).set_battery(state, battery.percentage / 2.55);
```

Comment:
I think this could provide a double as the percentage... would that be an issue?

Reply:
Yep, it should then automatically cast it back to the nearest int since it's the type of the param of the method. Not sure if there's a better way, we should be fine since we don't really care for the decimals and there's no need for a precise measure..